spec/price_policies_controller_shared_examples.rb

```diff
@@ -98,7 +98,7 @@
         end
         it 'should set fields based off the old policy' do
           do_request
-          assigns[:price_policies].map(&:price_group).should == [@price_policy.price_group, @price_group2_policy.price_group]
+          assigns[:price_policies].map(&:price_group).sort.should == [@price_policy.price_group, @price_group2_policy.price_group]
```

Comment:
If the version of rspec we're using supports it use `match_array` matcher instead of sorting.